settings.py

```diff
@@ -1452,7 +1452,7 @@ def _pkce_required(client_id):
         },
         'commcare_auth': {
             'handlers': ['file'],
-            'level': 'ERROR',
+            'level': 'INFO',
```

Comment:
Is this very chatty? If not, not problem, but if yes do you think it could be annoying when debugging other things?